programs/stake/src/stake_instruction.rs

```diff
@@ -6480,4 +6757,398 @@ mod tests {
             Err(StakeError::MinimumDelinquentEpochsForDeactivationNotMet.into()),
         );
     }
+
+    mod old_behavior {
+        use super::*;
+
+        fn new_feature_set() -> FeatureSet {
+            let mut feature_set = FeatureSet::all_enabled();
+            feature_set.deactivate(&feature_set::raise_minimum_stake_delegation_to_1_sol::id());
+            feature_set
+        }
+
+        #[test]
+        fn test_stake_process_instruction() {
+            do_test_stake_process_instruction(new_feature_set());
+        }
```

Comment:
You'll see that I now run basically all the tests twice—once for the old minimum delegation, and once for the new minimum delegation. I wanted to be extra sure I didn't break any existing behavior and also had the correct new behavior as well, so this was the best way I found to do that. It would've been nice to change the test's `process_instruction()` to just always do both, but since it returns accounts now, that wasn't feasible[^1].

[^1] I could've, by returning a tuple of accounts for old and new behavior, but then tests with multiple calls to process_instruction() wouldn't really be fixed.